engine/client/gl_rpart.c

```diff
@@ -609,7 +609,7 @@ set EF_BRIGHTFIELD effect
 */
 void GAME_EXPORT CL_EntityParticles( cl_entity_t *ent )
 {
-	float		angle;
+	// float		angle;
```

Comment:
Revert this to fix non-SSE build.

Reply:
What about using some kind of unused macro?
https://stackoverflow.com/questions/7090998/portable-unused-parameter-macro-used-on-function-signature-for-c-and-c

Reply:
It makes code difficult to read.
Unused variable warning will not break anything, it is just developer note, so it is better to leave variable as is. It is better to make coverity tests sometimes to search for dead code. Dead code much worse. It may break in future.

Reply:
@fuzun just ignore unused warning. Mostly, engine code does not have unused variables. I just disable it in compiler flags.

SSE defines outside mathlib isn't a good idea. I would fix it in new engine.